docs/source/app-dev/bindings-java/quickstart/template-root/src/main/java/com/daml/quickstart/iou/IouMain.java

```diff
@@ -65,23 +60,21 @@ public static void main(String[] args) {
         .getActiveContracts(ContractUtil.of(Iou.COMPANION), Collections.singleton(party), true)
         .blockingForEach(
             activeContracts -> {
-              activeContracts
-                  .getOffset()
-                  .ifPresent(offset -> acsOffset.set(new LedgerOffset.Absolute(offset)));
-              activeContracts
-                  .getContracts()
-                  .forEach(
-                      contract -> {
-                        long id = idCounter.getAndIncrement();
-                        contracts.put(id, contract.data);
-                        idMap.put(id, contract.id);
-                      });
+              activeContracts.offset.ifPresent(
+                  offset -> acsOffset.set(new LedgerOffset.Absolute(offset)));
+              activeContracts.activeContracts.forEach(
```

Comment:
maybe rename `activeContracts` to `response` so you get `response.activeContracts` here